vendors/twtg/drivers/neon-vs-qt-xx-xx/examples.json

```diff
@@ -0,0 +1,118 @@
+[
+    {
+        "description": "Payload for Neon Valve Sensor MT 1",
```

Comment:
YOu have used examples of VS-MT driver on a VS-QT driver.. is it only the description of the example which is wrong? if not, we have to remove these examples from this driver.

Reply:
The description was the problem. Examples seem still valid.